docs/_docs/tools/control-script.adoc

```diff
@@ -646,4 +646,136 @@ control.bat --change-tag _<new-tag>_
 ----
 --
 
+== Metric command
 
+Metrics command prints value(s) of the metric or metric registry from the parameter.
+
+If you need to get metric from the specific node, please, set the `--node-id` parameter.
+Without `--node-id` random node will be used.
+
+[tabs]
+--
+tab:Linux[]
```

Comment:
Could you do me a favor by replacing "Linux" with "Unix" for all the tabs on this page? That's my oversight. We need to use "Unix" since the commands are relevant for MacOS, Linux and other Unix-distributions.

Reply:
Done.